week-1/Homework/mandatory/2-logic-error.js

```diff
@@ -1,31 +1,31 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+    return word;
 }
 
 function getWordLength(word) {
-  return "word".length()
+    return word;
```

Comment:
In this function we want to get the word's **length** (quantity of characters in it). For example: 
if **word** is `hanif` the result would be `6`.

Same as above: You don't need to do it yourself, JavaScript has built in methods that you can use on strings. You can try looking for it on google and implement it here